portfolio/src/main/webapp/script.js

```diff
@@ -13,18 +13,45 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random quote to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['The only version of yourself that you should compare yourself to is past versions of yourself.',
+function addRandomQuote() {
+  const quotes =
+      ['The only version of yourself that you should compare yourself to is past versions of yourself. \n-Allison Raskin',
        'I am going my way and doing my best!',
-       ''];
+       "What I did each day would determine the kind of person I'd become. \n-Chris Hadfield"];
```

Comment:
Try to be consistent with your use of double- vs single- quotes. You used single quotes on line 20, but are using double quotes here. I don't think there's a universal standard for which one is "better", but you should try to be consistent with whichever one you pick.